assets/i18n/id.json

```
@@ -25,245 +25,243 @@
     "dgc-windows-wallet-modal-prosandcons": "Keuntungan dan Kerugian?", 
-    "dgc-windows-wallet-modal-different-strokes": "Masing - masing dompet memiliki keuntungan dan kerugiannya.", 
-    "dgc-windows-wallet-modal-multidoge-pro": "MultiDoge adalah sebuah light wallet, jadi itu hanya membutuhkan \"skim\" melalui blockchain sebelum dapat digunakan. Hal ini dilakukan dengan cepat.", 
-    "dgc-windows-wallet-modal-multidoge-con": "Tetapi dompet tersebut seharusnya hanya digunakan untuk penggunaan yang normal. Itu <strong>tidak dapat</strong> digunakan untuk menambang Dogecoin, karena dompet akan menjadi lambat dan tidak responsif jika digunakan untuk menambang Dogecoin.", 
-    "dgc-windows-wallet-modal-dogecoin-core-con": "Di sisi lain, Dogecoin Core mengunduh seluruh blockchain. Sinkronisasi awalnya jauh lebih lambat dibandingkan dengan MultiDoge, dan membutuhkan lebih banyak ruang.", 
-    "dgc-windows-wallet-modal-dogecoin-core-pro": "Tetapi ini adalah keuntungan karena aman untuk menambang Dogecoin dengan dompet ini, dompet ini dapat menangani transaksi dari pembayaran. Ini juga akan membantu menjaga jaringan Dogecoin.", 
+    "dgc-windows-wallet-modal-different-strokes": "Kedua dompet tersebut punya kelebihan dan kekurangan masing-masing.", 
+    "dgc-windows-wallet-modal-multidoge-pro": "MultiDoge merupakan dompet yang ringan, hanya butuh untuk \"membaca sepintas\" pada blockchain sebelum dapat digunakan. Proses ini berlangsung sangat cepat.", 
+    "dgc-windows-wallet-modal-multidoge-con": "Tapi dompet ini hanya digunakan untuk penggunaan yang wajar saja. Dompet tersebut <strong>tidak bisa</strong> digunakan untuk menambang Dogecoin, karena akan sangat lambat dan tidak responsif.", 
+    "dgc-windows-wallet-modal-dogecoin-core-con": "Sementara, Dogecoin Core mendownload keseluruhan data blockchain. Jadi, sinkronisasi awal agak lambat dibandingkan MultiDoge, dan memakan banyak ruang penyimpanan.", 
+    "dgc-windows-wallet-modal-dogecoin-core-pro": "Namun tentunya ada keuntungannya. Dogecoin jadi lebih aman dengan dompet ini, selain bisa melakukan transaksi, dompet ini juga menjaga jaringan Dogecoin tetap gesit.", 
     "dgc-osx-wallet-modal-dogecoin-core": "Dogecoin Core.", 
-	"dgc-android-wallet-modal-title": "UNDUH MELALUI GOOGLE PLAY.",
+	"dgc-android-wallet-modal-title": "UNDUH DARI GOOGLE PLAY.",
     "dgc-android-wallet-modal-apk": "<a href=\"https://github.com/langerhans/dogecoin-wallet-new/releases/download/1.08/wallet-1.08-aligned.apk\">Direct APK Download.</a>",
-	"dgc-ios-wallet-modal-title": "DOWNLOAD MELALUI THE APP STORE.",
+	"dgc-ios-wallet-modal-title": "UNDUH DARI APP STORE.",
     "dgc-getting-started-title": "MEMULAI",
-    "dgc-getting-started-desc": "MEMULAI DENGAN DOGECOIN",
-    "dgc-getting-started-is-123": "adalah <strong>semudah</strong> 1, 2, 3!",
+    "dgc-getting-started-desc": "Mulai menggunakan Dogecoin",
+    "dgc-getting-started-is-123": "itu <strong>Gampang Banget</strong>!",
```

I think since they add explanation how easy it is as easy as 1-2-3, do we need to add equivalent of it?

i.e. `Gampang Banget semudah membalikkan telapak tangan`. wdyt?

---

```suggestion
    "dgc-getting-started-is-123": "itu Gampang Banget <strong>semudah</strong> membalikkan telapak tangan!",
```
I've also moved the emphasis to match the en version, hope that's right.